sci-physics/rivet/rivet-3.1.10.ebuild

```diff
@@ -55,7 +56,6 @@ RDEPEND="
 "
 DEPEND="${RDEPEND}"
 BDEPEND="
-	virtual/fortran
```

Comment:
It does not require fortran? Or should we inherit `fortran-2.eclass` here?

Reply:
it does not have any fortran source code.